.aicoe-ci.yaml

```diff
@@ -10,4 +10,4 @@ deploy:
   project-org: open-services-group
   project-name: peribolos-as-a-service
   image-name: peribolos-service-controller
-  overlay-contextpath: "manifests/imagestream.yaml"
+  overlay-contextpath: "manifests/overlays/test/imagestream.yaml"
```

Comment:
Just a minor request.. since we're using this overlay as prod, what about we rename `test` to `prod`? WDYT @harshad16 ?

Reply:
sounds good to me , will make the change